src/schema/v2/auction_result.ts

```diff
@@ -37,6 +37,9 @@ export const AuctionResultSorts = {
       DATE_DESC: {
         value: "-sale_date",
       },
+      DATE_ASC: {
+        value: "-sale_date_asc",
```

Comment:
**Q:** Where is the value `-sale_date_asc` defined?

Reply:
I will be defining it in Diffusion in a bit 👀 
I am drafting the PR now

Reply:
Oki :)

Reply:
According to our usual conventions, ascending sale date would correspond with a `sale_date` value. This `-` and `...asc` seem to be in conflict.

Reply:
Nevermind! I see now this is updated in https://github.com/artsy/metaphysics/pull/4926.

Reply:
Thanks Joey 🙏 and @lordkiz